src/lib.rs

```diff
@@ -212,10 +212,7 @@ pub fn init_logging(opentelemetry_url: &Option<Url>) -> Result<(), LemmyError> {
     .with(ErrorLayer::default());
 
   if let Some(_url) = opentelemetry_url {
-    #[cfg(feature = "console")]
     telemetry::init_tracing(_url.as_ref(), subscriber, targets)?;
-    #[cfg(not(feature = "console"))]
-    tracing::error!("Feature `console` must be enabled for opentelemetry tracing");
```

Comment:
This is wrong, I believe it will crash if compiled with `--no-default-features`

Reply:
I added back the features and included the init of setting the defaults so that logs still work even if the otel setting is used.

Reply:
cfg only applies to the following line, so you need to wrap these two in `{}`, otherwise it will always log the error.

Reply:
Alright, sorry about that, I am not very knowledgeable in rust.